python_modules/dagster/dagster/_utils/error.py

```diff
@@ -244,8 +244,9 @@ def serializable_error_info_from_exc_info(
 
 
 DAGSTER_FRAMEWORK_SUBSTRINGS = [
-    "/site-packages/dagster/",
-    "/python_modules/dagster/dagster/",
+    "/site-packages/dagster",
```

Comment:
Do we worry about (1) third party dagster modules, e.g. `dagster-hightouch` and (2) users prefixing their own modules with `dagster-`? e.g. `dagster-mycompany`

Reply:
ya I thin thats a fair concern. What makes me feel ok is:
* third party libraries are still libraries for end users
* i would guess private `dagster-` prefixed modules to be quite rare
* being able to run with `--verbose` is a pretty low friction escape hatch